Documentation/Supported Mods.md

```diff
@@ -43,6 +43,12 @@ Custom NPCs:
 - [Custom NPC - Riley](https://www.nexusmods.com/stardewvalley/mods/5811), by SG, Version 1.2.2
 - [Alecto the Witch](https://www.nexusmods.com/stardewvalley/mods/10671), by zoedoll, Version 1.1.7
 
+In addition, a [Content Patcher mod](https://github.com/Witchybun/SDV-Randomizer-Content-Patcher) may be required for the mod to properly function alongside your game.  Include these alongside StardewArchipelago.  So far these are:

```

Comment:
I would suggest making this a link to your releases page within the github, for ease of use